src/components/Table/Table.test.ts

```diff
@@ -35,6 +35,18 @@ describe('template', () => {
     componentSlotRenderTest(Table, 'header-id', {props});
 
     componentSlotRenderTest(Table, 'id', {props});
+
+    componentWrapperClassTest(Table, {hover: true}, 'table-hover');
+
+    componentWrapperClassTest(Table, {click: true}, 'table-click');
+
+    it('emits event on click table row', async() => {
+        const wrapper = shallowMount(Table, {props});
+
+        wrapper.vm.$emit('click:row', props.items[0]);
+
+        expect(wrapper.emitted('click:row')[0]).toEqual([props.items[0]]);
+    });
```

Comment:
This test is funny 🤣 
```suggestion
    it('emits event on click table row', async() => {
        const wrapper = shallowMount(Table, {props});
        
        // Emit some event manually.
        wrapper.vm.$emit('click:row', props.items[0]);

        // Assert that I have just emitted the event manually.
        expect(wrapper.emitted('click:row')[0]).toEqual([props.items[0]]);
    });
```
It should be something more like this:
```suggestion
    it('emits event on click table row', async() => {
        const wrapper = shallowMount(Table, {props});

        // Click on the row, or let the row emit a click

        expect(wrapper.emitted('click:row')[0]).toEqual([props.items[0]]);
    });
```

Reply:
Didn't know how to emulate click that's why I did that 😄